### PR TITLE
RR-612 Integration tests for prison movements

### PR DESCRIPTION
**Pull request by mattwills-moj-digital**
This PR adds integration tests to ensure that timeline events are listed in the expected order, including the recently added prison movement events.

For example, we would normally expect a `PRISON_ADMISSION` event, following by an `ACTION_PLAN_CREATED` one etc:

image

### CODE REVIEW

File: integration_tests/pages/overview/TimelinePage.ts [hunk 1]
```diff
@@ -35,25 +35,10 @@ export default class TimelinePage extends Page {
...
-    // is the first event in time.
-    // We expect the last event in the DOM to be the first event in time, which should be ACTION_PLAN_CREATED
-    this.lastTimelineEvent().should('have.attr', 'data-qa-event-type', 'ACTION_PLAN_CREATED')
-    // We expect the 2nd to last event in the DOM to be the 2nd event in time, which should be MULTIPLE_GOALS_CREATED
-    this.lastButOneTimelineEvent().should('have.attr', 'data-qa-event-type', 'MULTIPLE_GOALS_CREATED')
-    return this
-  }
+  hasTimelineEventsInOrder = (events: string[]) =>
+    cy.get('div.moj-timeline div.moj-timeline__item').each((el, idx) => {
+      cy.wrap(el.attr('data-qa-event-type')).should('eq', events[idx])
```
Comment (mattwills-moj-digital) [inline integration_tests/pages/overview/TimelinePage.ts:40]: Credit to @nathanrussell-moj-digital for this. Not my area of expertise! 🤯

File: integration_tests/e2e/overview/timeline.cy.ts [hunk 2]
```diff
@@ -41,7 +41,13 @@ context('Prisoner Overview page - Timeline tab', () => {
     timelinePage //
       .activeTabIs('Timeline')
       .hasTimelineDisplayed()
-      .hasLearningAndWorkProgressPlanEventWithOneGoalDisplayed()
+      .hasTimelineEventsInOrder([
```
Comment (nathanrussell-moj-digital) [inline integration_tests/e2e/overview/timeline.cy.ts:44]: Love this syntax ❤️

File: integration_tests/pages/overview/TimelinePage.ts (hunk 1 above)
Comment (nathanrussell-moj-digital) [reply integration_tests/pages/overview/TimelinePage.ts:40]: Too kind! It's not my area of 'expertise' either. I might have had a little more exposure to things like the browser DOM (I wouldn't call that 'expertise' 😁 ), but the cypress API that encompasses that is mainly voodoo to me as well! 🤣 

In other news , the only thing I'd perhaps do to this method is declare a return type of `TimelinePage`, and then do `return this` at the end.
That way we can chain these method calls 👍

Comment (mattwills-moj-digital) [reply integration_tests/pages/overview/TimelinePage.ts:40]: Ah yes, good point. I had that in my original function but forgot to add it to this one!

**Review by nathanrussell-moj-digital — DISMISSED** [review]
LGTM 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**